src/precice/config/ParticipantConfiguration.cpp

```diff
@@ -549,10 +549,13 @@ void ParticipantConfiguration::finishParticipantConfiguration(
     io::PtrExport exporter;
     if (exportContext.type == VALUE_VTK) {
       if (context.size > 1) {
+        PRECICE_WARN("You use the VTK exporter in a parallel participant. Note that this will export as PVTU instead. For consistency, prefer to use \"<export:vtu ... />\" instead.");
```

Comment:
Minor language suggestions:

```suggestion
        PRECICE_WARN("You are using the VTK exporter in a parallel participant. Note that this will export as PVTU instead. For consistency, prefer \"<export:vtu ... />\" instead.");
```